docker/cassandra-stress/Dockerfile-src

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:8 as builder
+
```

Comment:
We are not really using this dockerfile... 

It's here only as a mean to build specific c-s out of the source code as needed.

In most cases we are going to be running from scylla docker images.

In general when not using our prepared VM, we don't need this copy.

So we might be better with just removing it

It was a workaround to disable logging, that was enabled by default on the code built in the loader VM images.

Reply:
So, is it correct to say that it is not needed in case of k8s/docker?

Reply:
I think, need to test to verify.

Reply:
We can change the log on this one, to be enable only when running with VMs directly (we are adding this option as a fallback for performance test, if it would be needed)